### ISSUE TITLE
cv-simd-shufflei* incorrect builtin

### ISSUE THREAD

**Issue by MaryBennett** [openhwgroup/corev-gcc#68]
Definition:
```
cv.shuffleI0.sci.b  rD,rs1,Is2   ;; flgs[7:6] = 0
```

Current implementation:
```
cv.shuffleI0.sci.b  rD,rs1,Is2   ;; Is2 = 0
```

Fix required:
 * Print Is2 as a 6-bit unsigned immediate, but match an 8-bit  unsigned immediate.
 * Change the predicates to check for 8-bit constant.
 * Change the constraint CF* to check bits [7:6]?
 * Change testcases.
   * cv-xcvsimd-march-compile-1.c
   * cv-simd-shufflei*-sci-b-compile-1.c

**Comment by jeremybennett** [comment]
Assigned to @MaryBennett to fix.

**Comment by MaryBennett** [comment]
Fixed in PR #72
